public/templates/partials/calendar/event/responses.tpl

```diff
@@ -73,4 +73,4 @@
       </div>
     </div>
   </div>
-</div>
```

Comment:
Please add the trailing newline back to the end of this file.

Reply:
It was added by this PR, is it missing on origin/master ? Looks like a Windows/Linux newline change to me.

Reply:
Ah, sorry. Misread the diff